dist/licensing/overrides.yaml

```diff
@@ -232,6 +232,36 @@
   url: http://twitter.github.com/bootstrap/javascript.html#transitions
   notice: Copyright (c) Twitter, Inc. (2012)
   license: ASL2
+
+- id: codemirror.{js,css}
```

Comment:
normal process is for the ID to refer to a *project* with a single entry and have a `files` line containing the files used, instead of the 5 different blocks for the same project; no biggie tho, i'll update and merge